docs/toc_en.yml

```diff
@@ -61,6 +61,10 @@ nav:
     - 'IPv4': 'data_types/domains/ipv4.md'
     - 'IPv6': 'data_types/domains/ipv6.md'
 
+- 'Database Engines':
+    - 'ClickHouse': 'database_engines/clickhouse.md'
+    - 'MySQL': 'database_engines/mysql.md'
```

Comment:
it seems like multilanguage symlinks are missing

Reply:
Yes, I didn't add the yet, I'll do.